.github/workflows/qa-testbench.yaml

```diff
@@ -0,0 +1,125 @@
+name: QA Testbench run
+
+on:
+  workflow_dispatch: {}
+  schedule:
+  # * is a special character in YAML so you have to quote this string
+    - cron:  '0 0 * * *'
+  
+inputs:
+  generation:
+    description: 'Specifies the generation template which should be used by the testbench run'
+    required: false
+    default: 'Zeebe SNAPSHOT'
+  branch:
+    description: 'Specifies the branch, for which the QA Testbench run should be executed'
+    default: 'main'
+    required: false
+  
+jobs:     
+  qa-testbench:
+    name: QA Testbench run
+    runs-on: ubuntu-latest
+    env:
+      IMAGE: "gcr.io/zeebe-io/zeebe"
+      GENERATION_TEMPLATE: "${{ github.event.inputs.generation }}"
+      BRANCH_NAME: "${{ github.event.inputs.branch }}"
+      ZEEBE_AUTHORIZATION_SERVER_URL: "https://login.cloud.ultrawombat.com/oauth/token"
+      BUILD_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
+    steps:
+      # Dynamic environment variables are not supported by GHA
+      # https://brandur.org/fragments/github-actions-env-vars-in-env-vars
+      - id: set-env
+        name: Set environment variables
+        run: |
+          echo 'TAG=SNAPSHOT-${GITHUB_SHA::8}' >> $GITHUB_ENV
```

Comment:
This is different to https://github.com/camunda/zeebe/blob/main/Jenkinsfile#L360 because I was not sure whether it really necessary to have the current snapshot version here. :shrug: